clearwater-memcached/usr/share/clearwater/infrastructure/scripts/memcached

```diff
@@ -25,4 +25,6 @@ sed -e 's/^-l .*$/-l '$listen_address'/g'\
     -e 's/^# *-v *$/-v/g'\
     -e 's/^\(# *\|\)-c.*$/-c 4096/g' </etc/memcached.conf >/etc/memcached_11211.conf
 
+echo "\n# Disable listening on UDP\n-U 0" >> /etc/memcached_11211.conf
+
```

Comment:
Also, clearwater infrastructure gets restarted a lot so it needs to be idempotent. This isn't and will result in loads of lines being added over time. I'm not sure if that is a problem for memcached, but it certainly looks messy. Take a look at these utilities that we use to add sections to config files: https://github.com/Metaswitch/clearwater-infrastructure/blob/master/clearwater-infrastructure/usr/share/clearwater/infrastructure/install/common